servant-docs/src/Servant/Docs/Internal.hs

```diff
@@ -959,6 +962,25 @@ instance (KnownSymbol sym, ToParam (QueryFlag sym), HasDocs api)
           paramP = Proxy :: Proxy (QueryFlag sym)
           action' = over params (|> toParam paramP) action
 
+-- | The docs for a @'QueryParamForm' a'@
+--   require a 'ToSample a' instance
+instance (ToForm a, ToSample a, HasDocs api)
```

Comment:
same here

Reply:
I went for re-export. It seemed like a straightforward solution.